packages/suite-web/e2e/tests/metadata/interval-fetching.test.ts

```diff
@@ -30,8 +30,15 @@ describe('Metadata - suite is watching cloud provider and syncs periodically', (
             cy.task('startEmu', { wipe: true });
             cy.task('setupEmu', { mnemonic: 'mnemonic_all' });
             cy.task('startBridge');
+            cy.prefixedVisit('/', {
+                onBeforeLoad: win => {
+                    cy.stub(win, 'open').callsFake(stubOpen(win));
+                    cy.stub(win, 'fetch').callsFake(rerouteMetadataToMockProvider);
+                },
+            });
+            cy.getTestElement('@welcome-layout/body', { timeout: 60_000 });
```

Comment:
why exactly was this change needed?

Reply:
For some reason `cy.clock()` was interfering with the discovery using core in module. So moving the discovery to happen before `cy.clock()` is initiated was fixing it. The time out should be `30_ 000`, I only changed it  to `60_000` during development, by I don't think it is needed so I changed it here https://github.com/trezor/trezor-suite/pull/14959/commits/99e500089ebd82c24a0f75f20dd5c59397a9e413

Reply:
Its probably because connect in iframe used to be isolated context, so cy.clock did not affect timers within it.

Reply:
Yes, something like that sounds like the case.